infoextract.js

```diff
@@ -29,6 +29,26 @@ module.exports = {
 		}
 	},
 
+	getChanneInfoYoutube(link) {
```

Comment:
Don't pass in the full link here, instead pass in the channel ID.

Reply:
I didn't pass in just the channel id in order to support links to users as well. If you look a few lines down you can see this when I parse the link to find if it starts with 'channel' or 'users'

Reply:
Pass in an object with either `channel` or `user` set.

Reply:
That sounds like a better idea. A channel url looks like .../channel/${channel_id}, where a user url looks like .../user/${youtube_username}